app/views/images/_image.html.erb

```diff
@@ -0,0 +1,55 @@
+<%
+  actions = []
```

Comment:
There is a lot of code in these views. Have you considered moving some of it out into a Helper class?
govuk_publishing_components does something similar for the step-by-step component: https://github.com/alphagov/govuk_publishing_components/blob/8b575b247e1eeaa0b3bf8544492fe4303de050d2/app/views/govuk_publishing_components/components/_step_by_step_nav.html.erb#L17